bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -936,6 +992,18 @@ void ReplicaImp::startConsensusProcess(PrePrepareMsg *pp, bool isCreatedEarlier)
     sendPartialProof(seqNumInfo);
   }
 }
+PrePrepareMsg *ReplicaImp::getConsensusPPFromDataPP(PrePrepareMsg *pp) {
```

Comment:
why not push the dataPP and its hash in the handler itself?
it would be easier to debug I think...